tests/bluetooth/tester/src/audio/btp/btp_pbp.h

```diff
@@ -0,0 +1,46 @@
+/* btp_pbp.c - Bluetooth PBP Tester */
+
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* This is main.h */
+
+#define BTP_PBP_READ_SUPPORTED_COMMANDS	0x01
+struct btp_pbp_read_supported_commands_rp {
+	uint8_t data[0];
```

Comment:
data[]  (I think we should no longer introduce new GNUisms)

Reply:
This gave a compiler warning if I did not add the zero size, if I remember correctly

Reply:
ahmmm   flex array cannot be only member in array... OK, we can sort it out later on (as this is same for all BTP services)